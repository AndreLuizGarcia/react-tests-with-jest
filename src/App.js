import React, { Component } from 'react'
import { Provider } from 'react-redux'
import store from './store'
import Todos from './Todos'

class App extends Component {
  render(){
    return(
      <Provider store={store}>
        <div className="App">
          <h2>Hello World</h2>
          <Todos />
        </div>
      </Provider>
    )
  }
}

export default App;
