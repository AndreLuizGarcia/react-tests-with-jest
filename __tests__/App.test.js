import React from  'react'
import App from '../src/App'
import renderer from 'react-test-renderer'

describe('Testing App Component', () => {
  it('render correctly', () => {
    const wrapper = renderer
      .create(<App />)
      .toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})