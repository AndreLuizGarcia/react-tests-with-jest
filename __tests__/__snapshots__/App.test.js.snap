// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Component render correctly 1`] = `
<div
  className="App"
>
  <h2>
    Oii
  </h2>
</div>
`;
